lib/components/artist/header.js

```diff
@@ -57,16 +57,31 @@ class Header extends React.Component {
         <Headline style={[styles.base, styles.headline]}>
           {artist.name}
         </Headline>
-        <SerifText style={styles.base}>
-          {artist.nationality}, b. {artist.birthday}
-        </SerifText>
+        {this.renderSection(artist.nationality, artist.birthday)}
```

Comment:
Until we extract functions from components (#113), we’ve been following the style to just access `this.artist` from the `renderSection` method.